recipes/cspice/all/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.1)
```

Comment:
this cmake file is compatible with https://github.com/microsoft/vcpkg/blob/master/ports/cspice/CMakeLists.txt :+1: